cmd/mothership/local/options.go

```diff
@@ -112,33 +113,66 @@ func componentsFromStrings(list []string, values []string) ([]*keb.Component, er
 	return comps, nil
 }
 
+func componentsFromClusterState(cluster cluster.State, values []string) ([]*keb.Component, error) {
```

Comment:
What does this function do? It creates components from cluster state, removes them, sorts them? Some verb is missing here

Reply:
The name of this function is consistent with other functions with similar responsibility (in the same go file) like `componentsFromFile` and `componentsFromStrings`